packages/pipeline/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import Pipeline from './Pipeline';
+import Stage from './Stage';
+import Counter from './Counter';
+import { Size, Variant } from './styles';
+
+export default Pipeline;
+
+export { Pipeline, Stage, Counter, Size, Variant };
```

Comment:
Sounds reasonable. I have opted for named export instead of default as the ideal usage would be to use the `Pipeline` component with the `Stage` component as children, so you would need a named export the later anyway.

Resolved by 45a95ac.